portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,44 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div>
+    <h1>Apoorva Dhanala's Portfolio</h1>
+    <img src="/images/me.jpg" align="right" alt="Here's a picture of me!" width="200" height="300">
```

Comment:
I think you might want images/me.jpg, right?

This is erroring out for me when I do  `google-chrome index.html` from ~/STEP-Portfolio/portfolio/src/main/webapp

Reply:
We were told to always start the path for images with "/" since the introductory walkthrough/its what was used in the sample code so I did that. You're talking about the / before images right?